utils.js

```diff
@@ -556,6 +556,12 @@ define(['jsunzip', 'promise'], function (jsunzip) {
             map(keyCodes.BACK_QUOTE, BBC.AT);
         }
 
+        // Swap APOSTROPHE and BACK_QUOTE key codes around for Mac users.
+        if (window.navigator.userAgent.indexOf("Mac") !== -1) {
```

Comment:
As noted elsewhere, maybe change to

```
if (window && window.navigator.....) {
```

(NB the space between the `if` and `(` too, just to be in keeping with the rest of the codebase.

Reply:
Cool, I'll get this done shortly.  Sorry for the slow reply, I've had a busy few days :)